policykit/templates/policyadmin/dashboard/dashboard_base.html

```diff
@@ -211,7 +211,8 @@ <h3 class="communityName">{{user.community.community_name}}</h3>
 
   <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
-  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+  <script src="{% static "bootstrap/js/bootstrap.min.js" %}"></script>
+  <script src="{% static "bootstrap/js/bootstrap-select.min.js" %}"></script>
```

Comment:
@amyxzhang I also made some performance improvements in this PR. We were previously loading some dependencies both from CDNs _and_ from local static files. I updated it to use local files for loading bootstrap and code-mirror. I also made it so we don't load code-mirror on the dashboard, which speeds things up. I changed the apache2 caching policy on policykit.org and policykit.metagov.org too, so things feel considerably faster!

Reply:
awesome!! Thank you!